client/src/assets/searchResult/index.js

```diff
@@ -0,0 +1,4 @@
+import hotel from './hotel.png';
+import restaurant from './restaurant.png';
+
+export default { hotel, restaurant };
```

Comment:
I don't like the `searchResult` directory, these images are not necessarily specific to search result and could be reused somewhere else. If you really want to keep them separate, pls rename the directory to sth else (looks more like a business domain/field/area)

Reply:
these images just fake Data

Reply:
sorry what do you mean?